pom.xml

```diff
@@ -71,6 +71,16 @@
 
   <dependencyManagement>
     <dependencies>
+      <dependency>
+        <!-- Versions conflict resolution. -->
+        <!-- support-core and others plugins require org.jenkins-ci.plugins:jackson2-api:2.11.0 -->
+        <!-- io.jenkins.configuration-as-code:test-harness:1.36 requires jackson 2.10.2 -->
+        <groupId>com.fasterxml.jackson</groupId>
```

Comment:
better to just exclude the test harness jackson

Reply:
⇒ #80